app/controllers/users_controller.rb

```diff
@@ -300,7 +300,7 @@ def reauthenticate
     else
       wrong_password!(params[:new_email],
                       t("users.confirm_change_email.wrong_password_html", contact_support_link: helpers.link_to(t("users.confirm_change_email.contact_support"), new_feedback_report_path)),
-                      t("users.changed_password.wrong_password"))
+                      t("users.changed_password.wrong_password_html", contact_support_link: helpers.link_to(t("users.confirm_change_email.contact_support"), new_feedback_report_path)))
```

Comment:
Please use a locale key for the link caption that is separate from the other error message, e.g. `users.changed_password.contact_support`